Script to export users from DocumentDB or MongoDB

mihaialdoiu opened the pull request:
*Issue #, if available:*

*Description of changes:*
Export Amazon DocumentDB users to user_output.js file, can be used to import them to other instance. Note: Passwords are not exported.

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

tmcallaghan reviewed (changes requested):
Couple of changes-
1. Add README.md
2. Change command line to accept --uri rather than individual parameters
3. What about users not in admin database?

mihaialdoiu commented:
Added Readme and changed the script to the same standard as the others in migration folder. Since MongoDB 2.6 all users are in admin db. I can add a note about support for DocDB 3.6+ and MongoDB 2.6+.
